### PR TITLE
Add case for not playing

### LINKED ISSUE #115: Handle Not Playing case in voiceCommandBlocked

### PR DESCRIPTION
**Pull request by 1Revenger1**
Closes #115

### CODE REVIEW

File: src/main/java/com/wylxbot/wylx/Core/Processing/ReactionProcessing.java [hunk 1]
```diff
@@ -60,21 +62,16 @@ private void removeRole(@NotNull GenericMessageReactionEvent event, Role roleToR
...
-            // TODO: Should probably do smth about this
-            return null;
-        }
-
-        User selfUser = event.getJDA().getSelfUser();
-
-        // Check if it's a reaction to a message Wylx sent
-        if (!msg.getAuthor().equals(selfUser)) {
-            return null;
+            // Couldn't get message, we should continue trying to get the menus anyways
```
Comment (joshuabrown419) [inline src/main/java/com/wylxbot/wylx/Core/Processing/ReactionProcessing.java:74]: will this cause any other errors? why do we need to get the message if its ok to proceed without it?

**Review by joshuabrown419 — CHANGES_REQUESTED** [review]

File: src/main/java/com/wylxbot/wylx/Core/Processing/ReactionProcessing.java (hunk 1 above)
Comment (1Revenger1) [reply src/main/java/com/wylxbot/wylx/Core/Processing/ReactionProcessing.java:74]: It's mostly to stop extra calls to the database for non existent reaction menus. I'm sort of wondering though if it really matters if we have to retrieve the message from discord though.

Comment (joshuabrown419) [reply src/main/java/com/wylxbot/wylx/Core/Processing/ReactionProcessing.java:74]: if we're just going to continue even if we arent able to retrieve it then it really doesnt make sense to try

**Review by joshuabrown419 — APPROVED** [review]
LGTM

**Pull request merged**